### PR TITLE
chore!: move community out of versioned docs

### PR DESCRIPTION
**Pull request by qweeah**
The PR provides an alternative to make community docs version-agnostic. Following the example of [Docusaurus multi-instance](https://docusaurus.io/docs/docs-multi-instance).

Merging this PR will have breaking changes on URL like `oras.land/docs/community/community_resources` and `oras.land/docs/next/community/community_resources` but can be fixed with extra refirection config on netlify side.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -103,6 +117,12 @@ const config = {
                         position: "left",
                         label: "Docs",
                     },
+                    {
+                        to: '/community/community_resources',
+                        label: 'Community Guide',
```
Comment (TerryHowe) [inline docusaurus.config.js:122]:
```suggestion
                        label: 'Community',
```

**Review by TerryHowe — COMMENTED** [review]
This looks good to me except this label

File: docusaurus.config.js (hunk 1 above)
Comment (qweeah) [reply docusaurus.config.js:122]: Doing it just to keep it same to the original label. `Community` sounds better in the top bar.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**